package.json

```diff
@@ -13,12 +13,12 @@
     "console": "truffle console",
     "coverage": "scripts/coverage.sh",
     "lint": "npm run lint:js && npm run lint:sol",
-    "lint:fix": "npm run lint:js:fix && npm run lint:sol:fix",
+    "lint:fix": "npm run lint:js:fix",
     "lint:js": "eslint .",
     "lint:js:fix": "eslint . --fix",
-    "lint:sol": "solium -d .",
-    "lint:sol:fix": "solium -d . --fix",
```

Comment:
I suppose solhint doesn't have a fix mode? Do you know if it is in their roadmap?

Reply:
I doesn't seem to have one, don't know about the roadmap. I've never found it particularly useful though.

Reply:
Hi guys. correct, Solhint doesn't have a 'fix' mode. [Solhint roadmap](https://protofire.github.io/solhint/roadmap.html) doesn't take into account this feature for the near future. But we can definitely include it in case there's enough interest from the community. We've just opened an [issue](https://github.com/protofire/solhint/issues/97) for it.